cinema/views.py

```diff
@@ -24,21 +31,29 @@
 class GenreViewSet(viewsets.ModelViewSet):
```

Comment:
You need to change inheritance. No need to use permissions for restrict (NotFound)